lib/openshift/cluster_logging.rb

```diff
@@ -100,7 +100,7 @@ def es_ready?(user: nil, cached: true, quiet: false)
 
     def wait_until_es_is_ready(user: nil, quiet: false, timeout: 10*60)
       success = wait_for(timeout) {
-        es_ready?(user: user, cached: false, quiet: quiet)
+        log_store_raw(user: user, cached: false, quiet: true ) && es_ready?(user: user, cached: false, quiet: true)
```

Comment:
I mean the `quiet` in `def wait_until_es_is_ready(user: nil, quiet: false, timeout: 10*60)`

We set the quiet defaults to true, but still allow people to set it to false when call the method if they want to show more logs specifically.